src/e2e/specs/dashboard.spec.ts

```diff
@@ -302,6 +302,58 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard  - Payment`, ()
       `${process.env.E2E_TEST_ENV === "prod" ? "yearly" : "every ⁨2⁩ months"}`,
     );
   });
+
+  test("Verify that the Premium upsell screen is displayed correctly - overview card", async ({
+    dashboardPage,
+    // purchasePage,
+    page,
+  }) => {
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463625",
+    });
+
+    //checking that the user can reach upsell page
+    await dashboardPage.goToDashboard();
+    const upsellButton = page.locator("section > div a");
+    await expect(upsellButton).toBeVisible();
+    await upsellButton.click();
+    await page.waitForURL(/.*\/fix\/.*\/view-data-brokers\/?/);
+    const removeThemButton = page.getByText("Remove them for me");
+    await removeThemButton.click();
+
+    //checking the bullet points
+    await page.waitForURL(/.*\/fix\/.*\/automatic-remove\/?/);
+    const ulElement = page.locator("div > strong + ul");
```

Comment:
Same comment as above, re: adding the custom locators to the page object model (POM) &mdash; if there isn't already an existing selector for this.

Reply:
In general I believe its best practice to use built-in playwright methods instead of xpaths. 

We could either use a more specific/readable xpath, like `//div[starts-with(@class, "dataBrokerProfiles_featuresList")]/ul`, or grab each li element with getByText() and assert their visibility in this case. 

I'm using [codegen](https://playwright.dev/docs/codegen) to generate playwright locators for me, and I notice theres some invisible char's in the text. This would cause `getByText(Monthly scan of 190 data)` to not work, which you might have tried. You would need to add the invisible char's by copying and pasting it directly from the site, `getByText(Monthly scan of ⁨190⁩ data broker)` (pasting this in vs-code will show the difference).

Reply:
@pdehaan same question as [here](https://github.com/mozilla/blurts-server/pull/4582#discussion_r1611874394) - do you think it should be in dashboard POM if these locators are multiple clicks away? I don't see a point in creating a separate POM just for these locators as well.